docs/api/cases/cases-api-create.asciidoc

```diff
@@ -0,0 +1,203 @@
+[[cases-api-create]]
+== Create case API
+++++
+<titleabbrev>Create case</titleabbrev>
+++++
+
+Creates a case.
+
+=== Request
+
+`POST <kibana host>:<port>/api/cases`
+
+=== Prerequisite
+
+You must have `all` privileges for the *Cases* feature in the *Management*,
+*{observability}*, or *Security* section of the
+<<kibana-feature-privileges,{kib} feature privileges>>.
+
+=== Request body
+
+`connector`::
+(Required, object) An object that contains the connector configuration.
++
+.Properties of `connector`
+[%collapsible%open]
+====
+`fields`::
+(Required, object) An object containing the connector fields.
++
+--
+To create a case without a connector, specify `null`. If you want to omit any
+individual field, specify `null` as its value.
+
+For {sn-itsm} connectors, refer to <<servicenow-action-configuration>>. For
```

Comment:
> For {sn-itsm} connectors, refer to <<servicenow-action-configuration>>. For

The problem with linking to those action pages is that they don't contain the exact keyword values that are required in these APIs. Therefore, I've left the bulleted lists here for now too. However, I think in the long run, the exact keyword values should be somewhere like the [create connector API](https://www.elastic.co/guide/en/kibana/master/create-connector-api.html). Ideally we'll be auto-generating these API docs soon enough to make that moot.

Reply:
Ah okay, so the fields I had commented out *do* work in this API? I had assumed they didn't since they weren't in the Actions docs.